### ISSUE TITLE
'error' event handler in 0.10

### ISSUE THREAD

**Issue by silverwind** [uWebSockets/uWebSockets#269]
I've been using handlers like these in 0.9:
```js
ws.on('message', fn1)
ws.on('close', fn2)
ws.on('error', fn3)
```
Since 0.10, there's an exception thrown when attempting to register the `'error'` event in above fashion:
````
Registering more than one listener to a WebSocket is not supported.
````
Is this intentional and if so, how would one access client-side errors in 0.10?

**Comment by silverwind** [comment]
Oh, and I also have a `wss.on('error', fn4)`, maybe the check is confusing these two handlers?

**Comment by alexhultman** [comment]
If you try the nodejs/client.js and add an error handler it works. Try and enter an invalid address and it will call the error handler. Can you try and do some debug prints and see what happens?

**Comment by alexhultman** [comment]
The removeListener is probably not implementing the 'error' string though. Are you relying on that?

**Comment by silverwind** [comment]
Working on a minimal example now, sec.

**Comment by silverwind** [comment]
No example yet, but I noticed that the node process starts saturating one CPU core on the first request, no matter if I try to attach that error handler or not. I'm attaching multiple websocket servers to a mix of http/https servers, will test further with less servers.

**Comment by alexhultman** [comment]
Sounds like something strange, will check this out tomorrow.

**Comment by silverwind** [comment]
Minimal example for the error handler issue:

```
"use strict";

var server = require("http").createServer(function(_, res) {
  res.end(`<!DOCTYPE html><head></head><body><script>
      var ws = new WebSocket("ws://" + document.location.host);
      ws.send("ping");
  </script></body>`);
});

server.listen(9000, "0.0.0.0", function() {
  var Wss = require("uws").Server;
  var wss = new Wss({server: server});
  wss.on("connection", function(ws) {
    console.log("ws connection");
    ws.on("message", console.log.bind(null, "message"));
    ws.on("close", console.log.bind(null, "close"));
    ws.on("error", console.log.bind(null, "error"));
  });
  wss.on("error", console.log.bind("wss error"));
});
```
Couldn't reproduce the CPU hang with it, but that one appears with just one server at least.

**Comment by alexhultman** [comment]
Ah okay I see. You are registering an 'error' to a server socket. Yeah that won't work, the server websockets do not have any error. This should be fixed so that it is ignored just like before.

**Comment by alexhultman** [comment]
This should be fixed in v0.10.3
